qiskit/test/mock/backends/__init__.py

```diff
@@ -15,6 +15,48 @@
 Mocked versions of real quantum backends.
 """
 
+# BackendV2 Backends
+from .almaden import FakeAlmadenV2
+from .armonk import FakeArmonkV2
+from .athens import FakeAthensV2
+from .belem import FakeBelemV2
+from .boeblingen import FakeBoeblingenV2
+from .bogota import FakeBogotaV2
+from .brooklyn import FakeBrooklynV2
+from .burlington import FakeBurlingtonV2
+from .cairo import FakeCairoV2
+from .cambridge import FakeCambridgeV2
+from .casablanca import FakeCasablancaV2
+from .essex import FakeEssexV2
+from .guadalupe import FakeGuadalupeV2
+from .hanoi import FakeHanoiV2
+from .jakarta import FakeJakartaV2
+from .johannesburg import FakeJohannesburgV2
+from .kolkata import FakeKolkataV2
+from .lagos import FakeLagosV2
+from .lima import FakeLimaV2
+from .london import FakeLondonV2
+from .manhattan import FakeManhattanV2
+from .manila import FakeManilaV2
+from .melbourne import FakeMelbourneV2
+from .montreal import FakeMontrealV2
+from .mumbai import FakeMumbaiV2
```

Comment:
I just remembered this will cause a conflict in the `__init__.py` for `qiskit/test/mock` because there is already a `MumbaiV2` class. I think we should probably rename the class `MumbaiV2` in https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/test/mock/fake_mumbai_v2.py to `FakeMumbaiFractionalCX` or something like that and just have an upgrade release note about the rename.

Reply:
Sure, should I create a separate PR or add on to this PR? Can you explain the meaning of "fractionalCX"?

Reply:
@mtreinish I submitted as a separate PR: https://github.com/Qiskit/qiskit-terra/pull/7839. Please let me know if you prefer to combine the changes to this PR.

Reply:
It's just the fractional application of a CX gate, the FakeMumbaiV2 is basically a target equivalent to mumbai with RZX gates tuned at fixed angles to experiment with the techniques from this paper: https://arxiv.org/abs/2111.02535

Doing it in this PR would have been a bit easier in this case just because of the release time rush, but in general it's better as a separate PR, I've went ahead and approved #7839.